pkgs/os-specific/linux/open-iscsi/default.nix

```
@@ -1,26 +1,26 @@
-{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod }:
+{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod, perl, systemd, pkgconf }:
 stdenv.mkDerivation rec {
   name = "open-iscsi-${version}";
-  version = "2.0-873-${stdenv.lib.substring 0 7 src.rev}";
+  version = "2.0-877-${stdenv.lib.substring 0 7 src.rev}";
```

I think we should remove the git revision from the name since releases have clear version numbers:
```suggestion
  version = "2.0-877";
```